Exercice_2_-_Le_chiffre_de_Cesar/caesar.py

```diff
@@ -80,83 +80,78 @@
 >>> chr(90+3)
 ']'
 """
-
+import string
 
 def do_cipher(plain_text: str, key: int) -> str:
-    # Met ton code ici
-    return "Il faudrait retourner un vrai truc ;)"
+    LIM_MIN_DOWN = ord('A')
+    LIM_MIN_UP = ord('Z')
+    ALPHABET_LOOP = len(string.ascii_uppercase)
+    
+    cipher_text = ""
```

Comment:
Comme tu verra plus tard dans un autre commentaire il vaut mieux que cipher_text soit une liste vide [] et pas une chaîne vide... wait and see